### 1. When the General Social Survey asked subjects whether they would be willing to accept cuts in their standard of living to protect the environment 344 of 1170 subjects said "yes."

#### A. Estimate the proportion in the population that would be expected to say "yes."

The proportion in the population that would be expected to say yes:

$\hat{p} = \frac{344}{1170} =0.294$

#### B. Conduct a sig. test to determine whether a majority or a minority of the population would say "yes." Report and interpret the value.

$H_{0}:$ $p = 0.50 vs. $H_{a}: $ $p \ne$ 0.5

SE =$(\frac{\hat{p}(1-\hat{p})}{n})^{1/2}$

Z= $\hat{p}$- $p_{0}$/SE

In [22]:
p_hat<-344/1170
p_hypothesized<-0.50
n<- 1170
SE<-sqrt(p_hat*(1-p_hat)/n)
Wald_test<-(p_hat-p_hypothesized)/SE
Wald_test
pnorm(-15.46)*2

[1] -15.46467

[1] 6.459853e-54

The Z-statistic is more than 3 standard devations away from the mean and the associated p value  6.46e-54 is < .05. Therefore, we have sufficient evidence to reject the null hyothesis that 50% of the population would say "yes".
Being that the sign of the z score is negative, we have evidence to support that it's the minority of the population that would say yes.

#### C. Construct and interpret a 99% conf. interval for the population proportion that would say "yes"

In [1]:
install.packages("binom")
library(binom)
w<-344
n<- 1170
alpha<-0.01
binom.confint(x=w,n=n,conf.level = 1 - alpha, methods ="asymptotic")

Updating HTML index of packages in '.Library'
Making 'packages.html' ... done


method     x   n    mean      lower     upper    
1 asymptotic 344 1170 0.2940171 0.2597081 0.3283261

Wald Confidence Interval: We are 99% confident that the true proportion that would be expected to say "yes" is between 0.26 and 0.33